test/parallel/test-cli-node-print-help.js

```diff
@@ -32,9 +32,9 @@ function validateNodePrintHelp() {
     { compileConstant: NODE_FIPS_MODE,
       flags: [ '--enable-fips', '--force-fips' ] },
     { compileConstant: NODE_HAVE_I18N_SUPPORT,
-      flags: [ '--experimental-modules', '--experimental-vm-modules',
-               '--icu-data-dir=dir', '--preserve-symlinks',
-               'NODE_ICU_DATA', 'NODE_PRESERVE_SYMLINKS' ] },
+      flags: [
+        '--icu-data-dir=dir', 'NODE_ICU_DATA', 'NODE_PRESERVE_SYMLINKS',
```

Comment:
nit: I'd rather be consistent with the existing entries in terms of style.

Reply:
Test still fails. Is `NODE_PRESERVE_SYMLINKS` still needed in this check?